__tests__/app.test.js

```diff
@@ -105,4 +105,56 @@ describe("/api/articles", ()=>{
       expect(body.articles).toBeSortedBy("created_at", { descending: true });
     });
   });
+})
+
+describe("/api/articles/:article_id/comments",() => {
+  test("200: Successful request to an API", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  })
+  test("200: responds with an array of correct length", () => {
+    return request(app)
+    .get("/api/articles/5/comments")
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.comments).toHaveLength(2);
+    })
+  })
+  test("200: should respond with array of comment objects and correct properties", ()=>{
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({ body }) => {
+      body.comments.forEach((comment) => {
+        expect(typeof comment.comment_id).toBe("number")
+        expect(typeof comment.body).toBe("string")
+        expect(typeof comment.article_id).toBe("number")
```

Comment:
We can be more specific with the article_id value here as we know what article_id we have requested.